Automattic/sensei
# Add border setting to Course List block

Fixes #5554.

### Changes proposed in this Pull Request
Another hacky hack:
- Add the border setting to the Query Loop block.
- Leave the border setting for the Course List block, but remove it for the Query Loop block.

Note that the theme has to declare support for block borders as well in order for the border setting to display. Astra and Blockwise are 2 themes that support block borders.

### Testing instructions
- Switch to the Astra theme.
- Add a Course List block and Query Loop block to a page or post.
- Switch between them and ensure you see a border setting for Course List, but not for Query Loop.
- Add a border to the block and ensure it works.

### Screenshot / Video
![Screen Shot 2022-09-02 at 3 45 50 PM](https://user-images.githubusercontent.com/1190420/188226152-56c92cc2-0a9b-4eec-a106-7ae3c419a13d.jpg)
